### PR TITLE
Orchestra signup

### PR DESCRIPTION
**Pull request by patriksletmo**
Added endpoints for orchestra signup and related functions.

### CODE REVIEW

File: app/controllers/api/v1/article_controller.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+class API::V1::ArticleController < ApplicationController
+  include ViewPermissionConcern
+
+  before_action :authenticate_user!
```
Comment (ljukas) [inline app/controllers/api/v1/article_controller.rb]: Varför en bang-metod?

File: test/models/special_diet_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class SpecialDietTest < ActiveSupport::TestCase
```
Comment (ljukas) [inline test/models/special_diet_test.rb]: SpecialDietTest

File: test/integration/orchestra_signup_integration_test.rb [hunk 3]
```diff
@@ -0,0 +1,124 @@
+require 'test_helper'
+
+class OrchestraSignupIntegrationTest < AuthenticatedIntegrationTest
+  test 'create orchestra' do
+    Orchestra.delete_all
```
Comment (ljukas) [inline test/integration/orchestra_signup_integration_test.rb]: Låter som man tar bort alla orchestrar här.

File: app/controllers/api/v1/article_controller.rb (hunk 1 above)
Comment (patriksletmo) [inline app/controllers/api/v1/article_controller.rb]: Required call to [devise_token_auth](https://github.com/lynndylanhurley/devise_token_auth), not corrected.

File: test/models/special_diet_test.rb (hunk 2 above)
Comment (patriksletmo) [inline test/models/special_diet_test.rb]: Corrected by removing empty test.

File: test/integration/orchestra_signup_integration_test.rb (hunk 3 above)
Comment (patriksletmo) [inline test/integration/orchestra_signup_integration_test.rb]: Corrected by adding a descriptive middle-man function including comments.

**Comment by patriksletmo** [comment]
Current version up and running on Heroku. Test the implementation by running `API_BASE_URL=https://sof-db-staging-pr-10.herokuapp.com rails s` from the branch `orkester-controller` in [sof-webapp](https://github.com/studentorkesterfestivalen/sof-webapp).

**Pull request merged**